src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -158,6 +158,16 @@ private void setIndentation(TerminalNode node) {
     indentation = Integer.parseInt(node.getText().substring(6));
   }
 
+  @Override
+  public void enterSimpleStatement(CirculationRulesParser.SimpleStatementContext ctx) {
+    ExprContext expr = ctx.expr();
+    Token token = ctx.getStart();
+    if (expr.criterium() != null && expr.policies() == null) {
```

Comment:
Is this expression only checks for a missing ':'? What if we create a rule that would have criteria and ':' but no policies, would this also be 'true'? I mean, something like `m book:`

Reply:
This case, you describe, is already handled  on 156 line CirculationRules.g4 and we have a test for that case on 480 line Text2DroolsTest.java So I am sure my code doesn't break this flow.